calico/network-policy/policy-rules/log-rules.mdx

```diff
@@ -0,0 +1,184 @@
+---
+description: Debug your policies with Log rules.
+---
+# Use Log action to debug policies
+
+## Big picture
+Calico Log rules are powerful resources that can be implemented in an environment to log the matching traffic and allowing you to determine how your policies are behaving.
+
+
+## Value
+Calico has a unique `Log` action that provides traffic observability and logging which is missing in the standard Kubernetes Network Policy. This unique action can be used by security teams and admins to troubleshoot their policies and make sure that their cluster security posture is doing what its expected to do.
+
+
+## Requirements
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs groupId='log-rules'>
+<TabItem label="eBPF" value="eBPF">
+    - Calico 3.29+
+    - Linux Kernel 5.16+
+
+    eBPF policy logs depend on advanced kernel features introduced in Linux Kernel 5.16 and later. If you encounter a scenario where logs only display the traffic verdict (e.g., ALLOWED or DENIED) without detailed information about IP packets, it indicates that your system is running on an older kernel version lacking the necessary capabilities.
+
+</TabItem>
+
+<TabItem label="iptables/ipvs" value="iptables">
+    - Rsyslogd
+    
+    :::note
+    
+    Keep in mind that `logs` action doesn't output anything if you are running a `Kind` cluster given that `rsyslogd` is not installed in `Kind` created clusters. 
```

Comment:
I'm going to replace it with
```
In a containerized Kubernetes environment, such as Kind, receiving syslog messages is not permittied. This limitation is by the design to prevent a noisy neighbor scenario since both containers and the host share the same Kernel .
```